ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/private/xpn/sdn/provision/cucushift-installer-rehearse-gcp-upi-private-xpn-sdn-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-private-xpn-sdn-provision
+  steps:
+  - ref: ipi-install-rbac
+  - ref: upi-conf-gcp-xpn
+  - ref: ignition-bastionhost
+  - ref: gcp-provision-bastionhost
+  - ref: proxy-config-generate
+  - chain: ipi-conf-gcp
+  - ref: sdn-conf
+  - ref: upi-conf-gcp
+  - ref: upi-install-gcp
+  - ref: ipi-install-times-collection
```

Comment:
Can  you help add `- ref: enable-qe-catalogsource` here.

Reply:
@jianli-wei I just checked several `cucushift-installer-rehearse-gcp-upi-private-xpn-provision`, did not append `- ref: enable-qe-catalogsource` there, can you help check if the existing workflow missing them, and file another PR to fix them.
The rules is connected env (also including behind proxy env) need `enable-qe-catalogsource` step, disconnected env need `enable-qe-catalogsource-disconnected` and `set-sample-operator-disconnected` step.